cache/remotecache/v1/spec.go

```diff
@@ -23,7 +23,7 @@ type LayerAnnotations struct {
 	MediaType string        `json:"mediaType,omitempty"`
 	DiffID    digest.Digest `json:"diffID,omitempty"`
 	Size      int64         `json:"size,omitempty"`
-	CreatedAt time.Time     `json:"createdAt,omitempty"`
+	CreatedAt time.Time     `json:"createdAt"`
```

Comment:
I'm guessing the removal of `omitempty` is because it is impossible for this to be empty?

Is there an intention for this to be nullable? It might be more correct to change this to:

```
CreatedAt *time.Time `json:"createdAt,omitempty"`
```

That would allow this to be nullable and `omitempty` to properly be present.

Reply:
I think in go1.24 these should be `omitzero` (unless it was just a bug and empty time should be in json)

Reply:
It seems this was more like a bug; I don't see a specific case where the value could logically actually be empty.

Reply:
I think in most cases, omit zero is what  omit empty should've been like in the first place. But yeah, it's a change in behavior (not sure if an important change for this one)